code/modules/cargo/packs.dm

```diff
@@ -1834,6 +1834,39 @@
 					/obj/item/vending_refill/hydronutrients)
 	crate_name = "hydroponics supply crate"
 
+/datum/supply_pack/organic/grill
+	name = "Grilling Starter Kit"
+	desc = "Hey dad I'm Hungry. Hi Hungry I'm THE NEW GRILLING STARTER KIT ONLY 5000 BUX GET NOW! Contains a grill and fuel."
```

Comment:
did you forget an "IT" between "GET" and "NOW"?

Reply:
no